File: server/src/main/java/by/brawl/ws/dto/SpellDto.java
package by.brawl.ws.dto;

import by.brawl.ws.holder.SpellHolder;

public class SpellDto {
    private String id;
    private Integer cooldownTurns;
    private Integer suspendTurns;

    public SpellDto(SpellHolder spellHolder) {
        id = spellHolder.getId();
        cooldownTurns = spellHolder.getCooldownTurns();
        suspendTurns = spellHolder.getSuspendTurns();
    }

    public String getId() {
        return id;
    }

    public Integer getCooldownTurns() {
        return cooldownTurns;
    }

    public Integer getSuspendTurns() {
        return suspendTurns;
    }
}

File: server/src/main/java/by/brawl/ws/dto/SpellDto.kt
package by.brawl.ws.dto

import by.brawl.ws.holder.SpellHolder

class SpellDto(spellHolder: SpellHolder) {

    val id = spellHolder.id
    val cooldownTurns = spellHolder.cooldown
    val suspendTurns = spellHolder.suspend

}